### PR TITLE
Add `button` to toggle music in `Let it snow` demo

### PR DESCRIPTION
**Pull request by alrra**
Fix MicrosoftEdge/Demos#210

### CODE REVIEW

File: letitsnow/js/default.js [hunk 1]
```diff
@@ -35,9 +36,31 @@
...
+        musicIsPlaying ? pauseMusic() : playMusic();
+    }
+
+    toggleSoundButton.addEventListener("click", toggleMusic);
+
     document.addEventListener("keypress", function (evt) {
+
         // do a secret move
-        if (evt.keyCode === secretCode) {
+        var charCode = (typeof evt.which == "number") ? evt.which : evt.keyCode;
```
Comment (sarvaje) [inline letitsnow/js/default.js]: Use `===` instead of `==`

File: letitsnow/js/default.js [hunk 2]
```diff
@@ -35,9 +36,31 @@
         backgroundGradientCanvas.height = window.innerHeight + 400;
     }
 
+    function pauseMusic() {
+         music.pause();
+         musicIsPlaying = false;
+         toggleSoundButton.innerHTML="Play music";
```
Comment (sarvaje) [inline letitsnow/js/default.js]: wrap the `=` symbol with white spaces

File: letitsnow/js/default.js [hunk 3]
```diff
@@ -35,9 +36,31 @@
...
+    }
+
+    function playMusic() {
+         music.play();
+         musicIsPlaying = true;
+         toggleSoundButton.innerHTML="Pause music";
```
Comment (sarvaje) [inline letitsnow/js/default.js]: wrap the `=` symbol with white spaces

**Review by sarvaje — CHANGES_REQUESTED** [review]

File: letitsnow/js/default.js (hunk 1 above)
Comment (alrra) [reply letitsnow/js/default.js]: @sarvaje Changes made!

**Review by sarvaje — APPROVED** [review]
LGTM

**Comment by sarvaje** [comment]
@melanierichards or @dstorey can you validate/merge this PR?

**Comment by melanierichards** [comment]
Fantastico, merging.

**Pull request merged**